package/xochitl/package

```diff
@@ -70,8 +70,15 @@ configure() {
     if [[ "x$(systemctl is-enabled sync.service)" != "xmasked" ]]; then
         systemctl mask sync.service
     fi
-    if ! is-active manual-sync.service; then
-        systemctl enable --now manual-sync.service
+
+    if [[ "x$(systemctl is-enabled rm-sync.service)" != "xmasked" ]]; then
```

Comment:
![image](https://github.com/toltec-dev/toltec/assets/1176979/bd180525-70f3-49fe-b424-0d06bd6fa768)
This will need to first check to see if the unit exists before checking if it's masked.

Reply:
We don't accept 'proceeding anyway'?

Reply:
> We don't accept 'proceeding anyway'?

No, as it adds noise that the end user doesn't need to care about, and there will be users who open tickets because of it.

Reply:
> `> /dev/null` which we already had in the postremove should fix this.

This will make it not display the error, but it will also hide real errors from the user. I'm not comfortable doing that. I'd much rather add a `unit-exists` method and check that before seeing if it's masked.

```bash
unit-exists() {
  systemctl --quiet list-unit-files "${1}" > /dev/null
}
```

Reply:
Upon testing on device, it looks like the version of systemctl that ships with it in 2.15 doesn't give failed error codes. This should work though:
```bash
unit-exists() {
  [ $(systemctl --quiet list-unit-files "${1}" | grep "${1}" | wc -l) -eq 1 ]
}

Reply:
#721 contains the unit-exists method, you can pull the implementation from there, or wait for it to be merged first.

Reply:
That's another way to accomplish it I guess